### DISCUSSION TITLE
Windows 10 - portable Zeal and VSCode

### DISCUSSION THREAD

**Discussion by Alsiri0n** [zealdocs/zeal#1573] [Q&A]
Hello, I updated Zeal to version 0.7.0 and facing with weird behaviour. Maybe you can help me solve my difficulty.
Explanation:
If I run Zeal from windows explorer and I don't have any problems - I saw my docsets.
But if I run Zeal from VSCode with help plugin - my Zeal is running, but my docsets are disapperaring.
And download button is disabled. If I select "Preference" menu and try to choose Docsets correct storage - I don't have any effect. My docsets won't appear.

**Comment by trollixx** [comment]
The issue caused by the current working directory (CWD) being set to something other than the actual folder where Zeal is installed. Not sure how to handle this best. It's a legitimate use case when application is launched with CWD different from where binary is located. Maybe at least detect it...
